src/app/garfile/departure/post.controller.js

```diff
@@ -7,7 +7,7 @@ const ValidationRule = require('../../../common/models/ValidationRule.class');
 
 const createValidationChains = (voyage) => {
   // Define port / date validation msgs
-  const portMsg = 'As you have entered a departure port code of "YYYY", you must provide longitude and latitude coordinates for the location';
```

Comment:
Redundant validation now, replaced with the validation for ensuring a choice is made.